pyspectral/atm_correction_ir.py

```diff
@@ -59,6 +59,7 @@
 
 
 import numpy as np
+import dask.array as da
```

Comment:
Could you maybe put this in a try/except block. If ImportError then `da = None`. You'd then need to update your `is_dask_data` boolean too.

Reply:
done